antarest/study/web/studies_blueprint.py

```diff
@@ -472,19 +470,16 @@ def output_download(
             extra={"user": current_user.id},
         )
         params = RequestParameters(user=current_user)
+        filetype = request.headers.get("Accept")
```

Comment:
you could still use an Enum with accepted types so that it will throw automatically a ValueError if the Accept header is not right.
```
class ExportFormat(str, Enum):
  ZIP="application/zip"
  TARGZ="application/tar+gz"
  ...

filetype = ExportFormat(request.headers.get("Accept"))
```